src/components/Alert.vue

```diff
@@ -11,6 +11,7 @@
     :border="border"
     :tile="tile"
     :dense="dense"
+    :colored-border="colored-border"
```

Comment:
How do you imagine Object property with dashed name? It will throws error. You should use camelCase name like coloredBorder